src/vue/forms/AtomicSwapQuoteAssetsForm.vue

```diff
@@ -178,15 +177,25 @@ export default {
               selectedSameAssetCode: (asset, quoteAsset) => !this.isAssetRepeated(asset.code, quoteAsset.type),
             },
             destination: {
-              required,
+              minValue: (value) => {
+                if (value.length > 0) {
+                  return required
+                } else {
+                  return true
+                }
+              },
```

Comment:
remove

Reply:
and remove `required`  import

Reply:
Removed minValue, i can't remove required import because it also use type and asset